docs/guides/data/storing_output.md

```diff
@@ -1,83 +1,87 @@
 #### Overview
 Saving information generated during a job is easy.
 
-On a FloydHub deep learning server your code has access to a directory called
-`/output`. The `/output` directory is a special directory that is used to store
-information you want to save for future use after a job finishes. Anything
-saved in the `/output` directory at the time a job finishes will be preserved
+Every artefacts(file, folder, dataset etc...) your code will save in the current working directory (`/floyd/home`), it will be returned in the **Files** tab of your Job page or from the Workspace's **File Viewer** when the workspace is not in *Running* state.
```

Comment:
All the files saved in the working directory (`/floyd/home`) will be preserved after the job or workspace finishes. In case of jobs, you will see this under the **Files** tab. For workspaces, you can open the workspace page to view the files.